types/parse-prefer-header/parse-prefer-header-tests.ts

```diff
@@ -0,0 +1,3 @@
+import parsePreferHeader = require('parse-prefer-header');
+
+parsePreferHeader('Prefer: code=500, value=50');
```

Comment:
those test could could probably be amended to cover:
```ts
parsePreferHeader(['respond-async, wait=100', 'handling=lenient']);
parse('')
```
as these seems to be supported input types:
https://github.com/ppaskaris/node-parse-prefer-header/blob/master/test.js
Support for `null`/`undefined` input is optional:
```ts
parse(null)
parse(undefined)
```
as probably not intended (but still tested in linked tests). Up to you with those last two.